backend/src/main/resources/application.properties

```diff
@@ -1,11 +1,11 @@
 spring.application.name=backend
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/db
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
This PRs scope is the frontend and its CI. Why do we need this change?
And on the other hand why the change? And how come it is not configurable.

Reply:
Well, no worries we can fix it later.

Reply:
Oh yeah, it ended up being a conflict, so all you have to do is just to resolve that.